xml/System.Windows.Media.Animation/Vector3DAnimation.xml

```diff
@@ -43,7 +43,7 @@
  For information about applying multiple animations to a single property, see [Key-Frame Animations Overview](~/docs/framework/wpf/graphics-multimedia/key-frame-animations-overview.md).  
   
 ## Freezable Features  
- Because the <xref:System.Windows.Media.Animation.Vector3DAnimation> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.Animation.Vector3DAnimation> objects gain several special features, which include the following: they can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ Because the <xref:System.Windows.Media.Animation.Vector3DAnimation> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.Animation.Vector3DAnimation> objects gain several special features, which include the following: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 Because the <xref:System.Windows.Media.Animation.Vector3DAnimation> class inherits from <xref:System.Windows.Freezable>, <xref:System.Windows.Media.Animation.Vector3DAnimation> objects gain several special features, which include the following: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```